OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -64,6 +64,9 @@
   <ItemGroup>
     <None Include="App.config" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
I'm using vstud 2013. Double checked to make sure it uses tabs and not spaces.

Strange it only did it for one file.